File: Java/JavaOO/ApostilaPOO/src/exerciciosPoo/exConta/Gerente.java
package exerciciosPoo.exConta;

public class Gerente extends Funcionario {
	int senha;
	int	numeroDeFuncionariosGerenciados;
	
	
	public Gerente(String nome, String cpf, double salario) {
		super(nome, cpf, salario);
	}
	
	
	@Override
	public double getBonificacao() {
		return super.getBonificacao() + 500;
	}
	

}

File: Kotlin/Kotlin/bytebank/src/main/kotlin/br/com/alura/bytebank/modelo/Gerente.kt
package br.com.alura.bytebank.modelo

import br.com.alura.bytebank.modelo.FuncionarioAdmin

class Gerente(
    nome: String,
    cpf: String,
    salario: Double,
    senha: Int
): FuncionarioAdmin(
    nome,
    cpf,
    salario,
    senha
){


    override fun bonificacao(): Double {
        return salario * 0.10 + salario
    }

}